src/components/MainContent/MainContent.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import CardFacts from "../CardFacts/CardFacts";
+
+const Wrapper = styled.body`
+  display: flex;
+  background: mintcream;
+  justify-content: center;
+  margin: 24px 0 24px 0;
+  height: 100vh;
+`;
+
+const MainContent = () => {
```

Comment:
Your components do not match the behavior described in the homework description

> If there is not enough main content - the `Footer` is displayed at the bottom, if there is a lot of content, then a `Footer` is visible only after fully scrolling to the bottom of the page.

You can change not only the `MainContent` but also other parts in the `App` file. Please change the layout of the components as described